vars/main.yml

```diff
@@ -1,2 +1,8 @@
----
-# vars file for template
\ No newline at end of file
+blivet_packages: {"RedHat-7": "python-blivet3",
+                  "RedHat-8": "python3-blivet",
+                  "Fedora": "python3-blivet",
+                  "CentOS-7": "python3-blivet",
```

Comment:
I suppose this is a mistake, CentOS-7 should be identical to Redhat-7, shouldn't it?

Reply:
fix in #33 - https://github.com/linux-system-roles/storage/pull/33/commits/375297db4d9874dadc1929831b5d2a7f3beae0c2